.travis.yml

```diff
@@ -2,13 +2,17 @@
 # based on http://anil.recoil.org/2013/09/30/travis-and-ocaml.html
 
 language: c
+sudo: required
 
-script:
-  - bash -ex .travis-ci.sh
```

Comment:
Since you're removing this line, you can probably delete this file as well.

Reply:
you are right !